### DISCUSSION TITLE
Icons not shown in the UI

### DISCUSSION THREAD

**Discussion by hellcry37** [home-assistant/frontend#11934] [Dashboard]
When you try to edit the icon of a device in Lovelace and start typing or scrolling in the icons field, no icons are shown until you select one.

I think this was introduced in 2022.3

2022.2.9 was ok or I did not notice this.

## System Health

version | core-2022.3.1
-- | --
installation_type | Home Assistant Container
dev | false
hassio | false
docker | true
user | root
virtualenv | false
python_version | 3.9.9
os_name | Linux
os_version | 5.14.0-051400-generic
arch | x86_64
timezone | Europe/Bucharest

<details><summary>Home Assistant Community Store</summary>

GitHub API | ok
-- | --
GitHub Content | ok
GitHub Web | ok
GitHub API Calls Remaining | 4932
Installed Version | 1.23.0
Stage | running
Available Repositories | 998
Downloaded Repositories | 11

</details>

<details><summary>Home Assistant Cloud</summary>

logged_in | false
-- | --
can_reach_cert_server | ok
can_reach_cloud_auth | ok
can_reach_cloud | ok

</details>

<details><summary>Lovelace</summary>

dashboards | 2
-- | --
resources | 7
views | 7
mode | storage

</details>

**Comment by zsarnett** [comment]
https://github.com/home-assistant/frontend/issues/11916
